### PR TITLE
Improve project language

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Remove usage of  the "blacklist" and "whitelist" whenever possible.

#### Rationale:

- Avoid using language that can be considered offensive

### CODE REVIEW

File: shared/checks/oval/no_cd_dvd_drive_in_etc_fstab.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
   <!-- If specified removable partition represents CD / DVD drive, create a variable
        holding also alternative names for CD / DVD block special device as used by udev -->
-  <constant_variable id="variable_cd_dvd_drive_alternative_names" datatype="string" comment="CD/DVD drive alternative names whitelist" version="1">
+  <constant_variable id="variable_cd_dvd_drive_alternative_names" datatype="string" comment="CD/DVD drive alternative allowed names" version="1">
```
Comment (evgenyz) [inline shared/checks/oval/no_cd_dvd_drive_in_etc_fstab.xml:19]:
```suggestion
  <constant_variable id="variable_cd_dvd_drive_alternative_names" datatype="string" comment="CD/DVD drive allowed alternative names" version="1">
```

File: shared/templates/mount_option_removable_partitions/oval.template [hunk 2]
```diff
@@ -27,7 +27,7 @@
 
   <!-- If specified removable partition represents CD / DVD drive, create a variable
        holding also alternative names for CD / DVD block special device as used by udev -->
-  <constant_variable id="variable_cd_dvd_drive_alternative_names_{{{ MOUNTOPTION }}}" datatype="string" comment="CD/DVD drive alternative names whitelist" version="1">
+  <constant_variable id="variable_cd_dvd_drive_alternative_names_{{{ MOUNTOPTION }}}" datatype="string" comment="CD/DVD drive alternative allowed names" version="1">
```
Comment (evgenyz) [inline shared/templates/mount_option_removable_partitions/oval.template:30]: Ditto.

File: products/ol8/profiles/ospp.profile [hunk 3]
```diff
@@ -161,7 +161,7 @@ selections:
     - auditd_freq
     - auditd_name_format
 
-    ### Module Blacklist
+    ### Module Disabled
```
Comment (yuumasato) [inline products/ol8/profiles/ospp.profile:164]: @ComplianceAsCode/oracle-maintainers Hi, are you okay with changes like these?

**Review by ggbecker — APPROVED** [review]

File: products/ol8/profiles/ospp.profile (hunk 3 above)
Comment (freddieRv) [reply products/ol8/profiles/ospp.profile:164]: No issue on our side 👍🏻

**Review by freddieRv — APPROVED** [review]

**Comment by yuumasato** [comment]
Merging, as it was approved by two parties, :)

**Pull request merged**